src/openlcb/AliasAllocator.cxx

```diff
@@ -89,15 +91,65 @@ void AliasAllocator::return_alias(NodeID id, NodeAlias alias)
         if_can()->frame_write_flow()->send(b);
     }
 
-    // This is synchronous allocation, which is not nice.
+    add_allocated_alias(alias);
+}
+
+void AliasAllocator::add_allocated_alias(NodeAlias alias)
+{
+    /** @TODO(balazs.racz): We leak aliases here in case of eviction by the
```

Comment:
The correct syntax is @<!-- -->todo for Doxygen.

Reply:
Done -- actually this todo is obsolete. Replaced it with a comment explaining why.